config/test.json

```diff
@@ -49,5 +49,17 @@
   "logging": {
     "level": "error",
     "fmt": "pretty"
-  }
+  },
+  "serviceClients": [
+    {
+      "id": "d23dbf62b82eb04e",
+      "name": "Test Service Client",
+      "scope": "profile:email",
+      "pubkey": {
+        "kty": "RSA",
+        "n":"xaQHsKpu1KSK-YEMoLzZS7Xxciy3fsGrhrrqW_JBrq3IRmeGLaqlE80zcpIVnStyp9tbet2niYTemt8ug591YWO5Y-S0EgQyFTxnGjzNOvAL6Cd2iGie9QeSehfFLNyRPdQiadYw07fw-h5gweMpVJv8nTgS-Bcorlw9JQM6Il1cUpbP0Lt-F_5qrzlaOiTEAAb4JGOusVh0n-MZfKt7w0mikauMH5KfhflwQDn4YTzRkWJzlldXr1Cs0ZkYzOwS4Hcoku7vd6lqCUO0GgZvkuvCFqdVKzpa4CGboNdfIjcGVF4f1CTQaQ0ao51cwLzq1pgi5aWYhVH7lJcm6O_BQw",
+        "e":"AQAB"
```

Comment:
I might not be following this closely enough to dive in here, but regarding the auth-server an JKUs, here's how it works:

@seanmonstar is the route you're talking about `/.well-known/public-keys`? If so, that's basically a "demo" route to show what a jku route should return and not used by preVerifyTokens (PVT) at all. 

What PVTs do use is a config list of `trustedJKUs` which get fetched as needed when specified in a PVT instead of having the jwks in config. In practice we only had one trusted JKU, marketplace, but it allowed for them to manage their keys on their servers without us being involved once we knew their JKU endpoint. If they ever changed them, we'd notice that we're missing a key with that id from the PVT and refetch. I hope that makes sense.

The key benefit is that we only needed to maintain the trusted JKU list, and not the actual keys.